### PR TITLE
Fix DeflateStream Performance test

### PR DESCRIPTION
**Pull request by Vedin**

### CODE REVIEW

**Comment by stephentoub** [comment]
@Vedin, can you briefly explain what the bug was and how this fixes it?

cc: @ianhays

**Comment by Vedin** [comment]
Right now, we enter the while (retCount != 0) loop once. After you break (after first iteration of benchmark), retCount will be 0 and you will not enter the loop again for subsequent Benchmark.Iterations.
I have just put it inside the loop

File: src/System.IO.Compression/tests/Performance/Perf.DeflateStream.cs [hunk 1]
```diff
@@ -97,22 +97,24 @@ public enum CompressionType
             NormalData = 4
         }
 
-        [Benchmark]
+        private const int Iter = 1000;
+
+        [Benchmark(InnerIterationCount = Iter)]
```
Comment (ianhays) [inline src/System.IO.Compression/tests/Performance/Perf.DeflateStream.cs]: This is reducing the number of inner iterations from 10000 to 1000. Is that on purpose, and if so, why?

**Comment by Vedin** [comment]
Because If on 10000 inner iterations we'll have only 1 Iteration of Benchmark, which doesn't show as an ok average time. So I reduced it to have more than 1 iteration of Benchmark and get a stable results.
image

File: src/System.IO.Compression/tests/Performance/Perf.DeflateStream.cs (hunk 1 above)
Comment (Vedin) [inline src/System.IO.Compression/tests/Performance/Perf.DeflateStream.cs]: Below

**Pull request merged**